server/server.js

```diff
@@ -11,3 +11,26 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+
+// import modules
+var http = require('http');
+var trucks = require('./trucks')
+var enhancedDate = require('./enhancedDate')
+
+// start a server
+http.createServer(function (request, response) {
+
+	// Access day name as a string using enhancedDate
```

Comment:
Missing the message specified in step 3: "Today is [day name], [month name] [date]. The food trucks available are :"